supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -387,39 +338,15 @@ With the exception of "sysprep file", which has a specific function, use "syspre
 
 *See also*:
 
-[discrete]
-[[tree-pane]]
-==== image:images/yes.png[yes] tree pane (noun)
-*Description*: The collapsible hierarchy of resources under *System* on the left-hand side of the Administration Portal. For a visual example, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.1/html-single/introduction_to_the_administration_portal/#Graphical_User_Interface_elements[Graphical User Interface Elements] in the _Introduction to the Administration Portal_. The screen shot applies to Red Hat Virtualization 4.1 and earlier versions (including 3.x).
-
-*Use it*: yes
-
-*Incorrect forms*: System pane, system pane
-
-*See also*:
-
-[discrete]
-[[user-portal]]
-==== image:images/yes.png[yes] User Portal (noun)
-*Description*: The User Portal is a graphical user interface provided by the Red Hat (Enterprise) Virtualization Manager in versions 4.1 and earlier. It has limited permissions for managing virtual machine resources and is targeted at end users.
-
-Always use "User Portal", including the capital P. Do not use in Red Hat Virtualization 4.2 and later, where the User Portal was replaced by the VM Portal.
-
-*Use it*: yes
-
-*Incorrect forms*: userportal, user portal, User portal, VM Portal
-
-*See also*: xref:administration-portal[Administration Portal], xref:vm-portal[VM Portal]
-
 [discrete]
 [[vm-portal]]
 ==== image:images/yes.png[yes] VM Portal (noun)
-*Description*: The VM Portal is a graphical user interface provided by the Red Hat Virtualization Manager in versions 4.2 and later. It has limited permissions for managing virtual machine resources and is targeted at end users.
+*Description*: The VM Portal is a graphical user interface provided by the Red Hat Virtualization Manager. It has limited permissions for managing virtual machine resources and is targeted at end users.
 
-Always use "VM Portal", including the capital P. Do not use in Red Hat Virtualization 4.1 and earlier, where it did not yet exist; use "User Portal" instead.
+Always use "VM Portal", including the capital P.
```

Comment:
Instead of capital P, maybe something like |"Always use "VM Portal" and capitalize the product name." WDYT?